js/src/theme.js

```diff
@@ -232,7 +232,12 @@ const theme = createTheme({
     },
     MuiFormControlLabel: {
       root: {
-        width: 'max-content',
+        display: 'flex',
+        alignItems: 'flex-start',
+      },
+      label: {
+        paddingTop: '5px',
+        paddingBottom: '5px',
```

Comment:
@i-amdroid can you remind me when it's better to use pixels and when it's better to use built-in theme spacing?